common/types/src/pruntime.rs

```diff
@@ -30,7 +31,6 @@ where
     pub stake_received_kv: Vec<StorageKV<Balance>>,
     pub online_workers_kv: StorageKV<u32>,
```

Comment:
We can move `OnlineWorkerSnapshot` to pruntime, because it's not used in any other packages any more.

Btw, I think it makes sense to store `online_workers` and `compute_workers` directly as `u32`, and use HashMap or BTreeMap to replace these maps. The old implementation was designed in that way because we need to collect kv pairs and validate them in a StorageProof, which is not used now.

Reply:
done